frontends/mit-open/src/pages/FieldPage/FieldPage.tsx

```diff
@@ -32,6 +33,9 @@ const FieldPage: React.FC = () => {
     channelType && (
       <FieldPageSkeleton name={name} channelType={channelType}>
         <p>{fieldQuery.data?.public_description}</p>
+        {channelType === "offeror" ? (
+          <TestimonialDisplay offerors={[name]} />
+        ) : null}
```

Comment:
Could you rebase? (May be easier to squash and then rebase).

This should now be `channelType === "unit"`.